src/test/fuzz/proto/atmp.proto

```diff
@@ -0,0 +1,81 @@
+syntax = "proto2";
+
+package proto_fuzz;
+
+message PrevOut {
+  // Since we fill in the txid on transactions via post processing, we enable
+  // the fuzzer to copy those txids into PrevOuts.
+  required string txid = 1;
+  required uint32 n = 2;
+}
+
+message Script {
+  required bytes raw = 1;
+}
+
+message TxInput {
+  required PrevOut prev_out = 1;
+  required Script script_sig = 2;
+  required uint32 sequence = 3;
+
+  // scriptWitness
+  repeated bytes witness_stack = 4;
+}
+
+message TxOutput {
+  required int64 value = 1;
+  required Script script_pub_key = 2;
+}
+
+message Transaction {
```

Comment:
nit: Is it possible to de-duplicate proto sections (for example common stuff like a transaction)?

Reply:
Yes, afaik you we could have a sort of library of proto definitions that can be reused.